app/src/main/res/xml/pref_notifications.xml

```diff
@@ -115,6 +115,21 @@ S        <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/and
             </PreferenceCategory>
 
             <PreferenceCategory
+                android:title="Missed Readings">
```

Comment:
I guess this would double the missed readings settings as it is already there in line 95:
https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/256/files#diff-9abb79954cd4af4a8b02a7e89d92964bR95